.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: ruby
+os: osx
+env: OSX=10.12
```

Comment:
Given that we recommend building formulae in this repo on older versions of OS X, maybe we should test using an older release too.